spec/controllers/sys_controller_spec.rb

```diff
@@ -211,14 +212,14 @@ module OpenProjectRepositoryAuthenticationSpecs
 
       it 'should return the same as user_login for valid creds' do
         expect(controller.send(:cached_user_login, valid_user.login, valid_user_password)).to eq(
-        controller.send(:user_login, valid_user.login, valid_user_password)
-        )
+                                                                                                controller.send(:user_login, valid_user.login, valid_user_password)
+                                                                                              )
       end
 
       it 'should return the same as user_login for invalid creds' do
         expect(controller.send(:cached_user_login, 'invalid', 'invalid')).to eq(
-          controller.send(:user_login, 'invalid', 'invalid')
-        )
+                                                                               controller.send(:user_login, 'invalid', 'invalid')
```

Comment:
Line is too long. [129/100]<br>Indent the first parameter one step more than the previous line.